### PR TITLE
docs: Update Fether FAQ with what networks are supported

### PR DESCRIPTION
**Pull request by ltfschoen**

### CODE REVIEW

File: Fether-FAQ.md [hunk 1]
```diff
@@ -122,6 +122,18 @@ You can pass specific flags (such as `--chain`) for Fether to launch the underly
 $ /path/to/fether --chain ropsten --light
 ```
 
+Fether only supports networks that are compatible with running a light client node using the `--light` option of the Fether CLI that is shown above, and as mentioned in the FAQ [What networks are supported by Fether?](#what-networks-are-supported-by-fether?).
+
+### What networks are supported by Fether?
+
+Fether and Parity Signer only support networks that are compatible with running a light client, as explained in the FAQ [How to launch Fether on a different network?](#where-are-fether-logs-stored?).
```
Comment (Tbaut) [inline Fether-FAQ.md]: Parity Signer is not related to light clients and we shouldn't mention it here.
I think we shouldn't talk about light client networks anyways, some network probably have light clients but we don't support them officially (no ERC-20 token for instance).

```suggestion
Fether currently supports: 
- Ethereum mainnet
- Ethereum classic 
- Kovan testnet
- Ropsten testnet

Görli testnet support will be added soon.

Read here [how to launch Fether on a different network](#how-to-launch-fether-on-a-different-network).
```

File: Fether-FAQ.md [hunk 2]
```diff
@@ -122,6 +122,18 @@ You can pass specific flags (such as `--chain`) for Fether to launch the underly
...
+
+The networks that currently support running a light client node include:
```
Comment (Tbaut) [inline Fether-FAQ.md]: These lines should be removed if you accept my suggestion below.

**Comment by ltfschoen** [comment]
@Tbaut i've pushed commits addressing your comments

**Comment by Tbaut** [comment]
Top, thanks

**Pull request merged**